src/cli.ts

```diff
@@ -176,7 +176,7 @@ function networkFromOptions(options: OptionValues): string {
   return network
 }
 
-function getOptions(program: Command, options: OptionValues): OptionValues {
+export function getOptions(program: Command, options: OptionValues): OptionValues {

```

Comment:
please add documentation for all the exported function, interfaces, constants and enums